src/components/Textarea/Textarea.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {StoryObj, Meta} from '@storybook/react';
+
+import {Textarea} from './index';
+
+const meta: Meta<typeof Textarea> = {
+    title: 'Components/Textarea',
+    tags: ['new'],
+    component: Textarea,
+    decorators: [
+        StoryCmp => (
+            <div style={{width: '50vw'}}>
+                <StoryCmp/>
+            </div>
+        )
+    ],
+    parameters: {
+        layout: 'centered'
+    },
+    args: {
+        placeholder: 'Placeholder text'
+    },
+    argTypes: {
+        onChange: {action: 'onChange'},
+        onBlur: {action: 'onBlur'},
+        onFocus: {action: 'onFocus'}
+    }
+};
+export default meta;
+
+type Story = StoryObj<typeof Textarea>;
+
+export const Uncontrolled: Story = {
+    args: {
+        value: 'Uncontrolled textarea'
+    }
+};
+
+export const Controlled: Story = {
+    args: {
+        placeholder: 'Controlled textarea'
+    }
+};
```

Comment:
A story for a controlled component must use `value` and update it with the `onChange` function. In this example you still use the uncontrolled version of the component.